code/game/objects/structures/memorial.dm

```diff
@@ -8,7 +8,7 @@ but we are all contributors together.
 
 Goodbye, man. We'll miss you.
 
-This memorial has been designed for him and any future coders to perish.
```

Comment:
top kek

Reply:
keep this in i've been laughing for 2 minutes now

Reply:
fucking insensitive...